### PR TITLE
badge-component added

### PR DESCRIPTION
**Pull request by rishabh486**
pr for badge component

### CODE REVIEW

**Comment by rishabh486** [comment]
Screenshot (31)

**Review by developer-anand — CHANGES_REQUESTED** [review]
Use variables for colors do the changes

File: badges.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  height: 5rem;
+}
+.icon-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 4rem;
+  height: 4rem;
+  color: #333333;
+  background: lavenderblush;
```
Comment (rutu14) [inline badges.css]: Can use color variables

File: badges.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  width: 100%;
+}
+.icon-badge-number {
+  position: absolute;
+  top: 0;
+  right: 0rem;
+  width: 1.5rem;
+  height: 1.5rem;
+  background: lightblue;
+  color: black;
```
Comment (rutu14) [inline badges.css]: Define variables for colors and use them.

**Review by rutu14 — APPROVED** [review]

File: badges.css (hunk 1 above)
Comment (rishabh486) [reply badges.css]: changes made

File: badges.css (hunk 2 above)
Comment (rishabh486) [reply badges.css]: donw with the suggestion

**Pull request merged**